frontend/src/pages/ProjectDetails.tsx

```diff
@@ -250,14 +250,30 @@ const ProjectDetailsPage = () => {
             <div className="h-64 overflow-y-auto pr-2">
               {project.issues.map((issue, index) => (
                 <div key={index} className="mb-4 rounded-lg bg-gray-200 p-4 dark:bg-gray-700">
-                  <h3 className="font-semibold">{issue.title}</h3>
+                  <h3 className="font-semibold">
+                    <a
+                      href={issue.url}
+                      target="_blank"
+                      rel="none"
+                      className="text-[#1d7bd7] hover:underline dark:text-sky-600"
+                    >
+                      {issue.title}
+                    </a>
+                  </h3>
                   <div className="mt-2 flex items-center">
-                    <img
-                      src={issue.author.avatar_url}
-                      alt={issue.author.name}
-                      className="mr-2 h-6 w-6 rounded-full"
-                    />
-                    <span className="text-sm">{issue.author.name}</span>
+                    <a
```

Comment:
These are showing up as undefined for me right now:
![Screenshot 2025-01-22 at 6 53 30 PM](https://github.com/user-attachments/assets/fb67346d-f248-4dd6-9486-d73469c6f5f1)